Wireframe/index.html

```diff
@@ -1,13 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Git Essentials</title>
+  <link rel="stylesheet" href="./css/style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Orbitron:wght@400..900&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <!-- Add your HTML markup here -->
+  <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+
+  <header>
+    <h1>Version Control System</h1>
+    <p>Version control, also known as source control, is the practice of tracking and managing changes to software
+      code.
+    Version control systems (VCS ) are software tools that help software teams manage changes to source code over
+      time.
+    VCS tools have seen great improvements over the past few decades and some are better than others.
+      One of the most popular VCS tools in use today is called <strong><em>Git</em></strong>.
+    </p>
+  </header>
+  <main>
+    <article>
+      <div id="article-one-title">
+        <img src="./Images/Gitlogo.png" alt="Git logo" id="git-logo">
+        <h2 id="article-one-h2-margin">What is Git?</h2>
+      </div>
+      <p>
+        Git is a version control software that allows developers to create and manage different versions of a project. It is a free and open source distributed version control system designed to handle everything from small to very large projects with speed and efficiency.
+      </p>
+      <a href="https://www.atlassian.com/git/tutorials/what-is-git" target="_blank"><button class="btn-format">Read More</button></a>
```

Comment:
Nice. I see you used target="_blank" to open page in a new tab!

One issue here is that it's not good idea to put <button> tag inside <a> or vice versa. If you check https://validator.w3.org this will be suggested for you.
Instead try to use only <a>, and style it like a button in your css.